jupyterhub/app.py

```diff
@@ -3191,6 +3197,11 @@ def stop(self):
             self.http_server.stop()
         self.io_loop.add_callback(self.io_loop.stop)
 
+    async def start_show_config(self):
+        """Async wrapper around base start_show_config method"""
```

Comment:
Can you add a comment on why this is needed?

Reply:
👍 comment added